code/controllers/configuration/entries/game_option.dm

```diff
@@ -0,0 +1,117 @@
+
+//game_options.txt configs
+
+/datum/config_entry/number/health_threshold_dead
+    config_entry_value = -100
+    integer = FALSE
+
+/datum/config_entry/number/organ_health_multiplier
+    config_entry_value = 0.9
+    integer = FALSE
+
+/datum/config_entry/number/organ_regeneration_multiplier
+    config_entry_value = 0.25
+    integer = FALSE
+
+/datum/config_entry/flag/organs_decay
+
+//Paincrit knocks someone down once they hit 60 shock_stage, so by default make it so that close to 100 additional damage needs to be dealt,
+//so that it's similar to PAIN. Lowered it a bit since hitting paincrit takes much longer to wear off than a halloss stun.
+/datum/config_entry/number/organ_damage_spillover_multiplier
+    config_entry_value = 0.5
+    integer = FALSE
+
+/datum/config_entry/flag/bones_can_break
```

Comment:
I believe we had bone breaking disabled

Reply:
Config => game_options

Reply:
Right, but you can set a default option here can't you? Just in case something goes wrong with config

same for the other two comments